wordle-web/pages/game.vue

```diff
@@ -35,20 +35,52 @@
             <span> Go Home </span>
           </v-tooltip>
         </v-col>
+        <v-col cols="5" class="d-flex flex-row-reverse">
```

Comment:
You may consider making this into it's own component.

Reply:
Additionally, if the username is 'Guest' the game should prompt for a name at the end. (dialog should pop up before the save occurs.)